packages/app/src/pages/index.tsx

```diff
@@ -90,18 +92,30 @@ const Home: FCWithLayout<typeof getStaticProps> = (props) => {
       <Box bg="white" pb={4}>
         <MaxWidth>
           <TileList>
-            <WarningTile
-              message={siteText.regionaal_index.belangrijk_bericht}
-              variant="emphasis"
-            />
-
-            <Search />
-
             <TopicalPageHeader
               lastGenerated={Number(lastGenerated)}
               title={replaceComponentsInText(text.title, {
-                the_netherlands: <strong>{text.the_netherlands}</strong>,
+                the_netherlands: text.the_netherlands,
               })}
+              link={
+                <LinkWithIcon
+                  href="/landelijk/vaccinaties"
+                  icon={<ArrowIconRight />}
+                  iconPlacement="right"
+                  fontWeight="bold"
+                >
+                  {text.title_link}
+                </LinkWithIcon>
+              }
+            />
+
+            <Box width={{ _: '100%', lg: '75%' }}>
```

Comment:
width is 100% by default I think

```suggestion
            <Box width={{ lg: '75%' }}>
```